### PR TITLE
Update pdo-prepare.md

### PR DESCRIPTION
**Pull request by phanos**
Removing stray self-closing anchor tag

### CODE REVIEW

**Comment by phanos** [comment]
As it is now, entire "Prepare Example" section is rendered as a broken hyperlink.
image

**Comment by ShannonLeavitt** [comment]
@phanos - Your change removes an important bookmark. Rather than deleting it, can you just fix the syntax to something like this? `<a name="emulate-prepare"></a>`

@David-Engel 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
Thanks for your contribution, @phanos. However, I'm closing this because it breaks the existing anchor tag.

**Review by rwestMSFT — COMMENTED** [review]
Update

File: docs/connect/php/pdo-prepare.md [hunk 1]
```diff
@@ -157,8 +157,6 @@ $p = 'my data';
 $statement->bindParam(':myVarcharValue', $p, PDO::PARAM_STR, 0, PDO::SQLSRV_ENCODING_SYSTEM);
 ```
 
-<a name="emulate-prepare" />
-
 ## Prepare example
```
Comment (rwestMSFT) [inline docs/connect/php/pdo-prepare.md]:
```suggestion
## <a id="emulate-prepare"></a> Prepare example
```

**Comment by rwestMSFT** [comment]
I will fix this in an internal PR. Thanks for your feedback, @phanos.

**Comment by rwestMSFT** [comment]
@phanos The updated article will go live in a day or so. Thanks again for your feedback, and sorry for the confusion.

**Pull request closed without merging**